llpc/context/llpcPipelineContext.cpp

```diff
@@ -130,6 +130,11 @@ static cl::opt<int> DontUnrollHintThreshold("dontunroll-hint-threshold",
                                             cl::desc("loop unroll threshold to use for loops with DontUnroll hint"),
                                             cl::init(0));
 
+// -lds-spill-limit-dwords: Maximum amount of LDS space to be used for spilling. Only non-negative values are allowed,
+// the value of 0 disables LDS spilling.
+static cl::opt<int> LdsSpillLimitDwords("lds-spill-limit-dwords",
```

Comment:
If only non-negative numbers are allowed, why not make this unsigned like ldsSpillLimitDwords?